Podfile

```diff
@@ -169,6 +169,24 @@ target 'WordPress' do
         shared_test_pods
         pod 'Nimble', '~> 7.3.1'
     end
+
+    ## Convert the 3rd-party license acknowledgements markdown into html for use in the app
+    post_install do
+        require 'commonmarker'
+        
+        markdown = File.read('Pods/Target Support Files/Pods-WordPress/Pods-WordPress-acknowledgements.markdown')
+        rendered_html = CommonMarker.render_html(markdown, :DEFAULT)
+        styled_html = "<style>
+                         body {
+                           font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+                         }
+                      </style>
+                      <body>
+                        #{rendered_html}
```

Comment:
I had the wrong destination repo in my initial PR.  This is a comment by @jtreanor from that discussion: https://github.com/markpar/WordPress-iOS/pull/1#discussion_r287318728

Reply:
> Could we add a <title> here too so that it is displayed above the Web View?
 
That's a great suggestion!  We can do it in `Podfile`'s `post_install` hook, along with a quick call to `sub` to remove the existing `<h1>Acknowledgements</h1>`.  Here's a (snipped) screenshot:

![image](https://user-images.githubusercontent.com/6062009/58364461-859f8280-7e69-11e9-8f63-f4d1e1ed4f0d.png)

> Of course, this file isn't localized but considering the generated file isn't either, maybe that would be okay. Alternatively, we could add it in the Swift code and use NSLocalizedString.

I started looking into a way to make the title localizable and thought one way to do would be to first render a token (instead of the string `Acknowledgements`) in the `<title>` during `pod install`.  Then later, in Swift, load the HTML into memory and do a find & replace with the localized title string.  

At this point, we could do something like writing the in-memory HTML to a temp file on the device and then pass that URL to `displayWebView`.  This would work, but it feels a little more complicated than it needs to be.   

I found that `WKWebView` has a method `loadHTMLString`; however, it might involve touching `WebKitViewController` and/or `WebViewControllerConfiguration` to support it.  While I am always in favor of localization, this is the point where I would stop and ask others to weigh in on whether it merits further investigation.  🙂

Reply:
Thanks for digging into the details here.

Let's start simple by adding the `<title>` in `post_install` without localization as you mention and explore those other approaches if we feel there is a need.

Reply:
👍 That sounds like a great plan.